### ISSUE TITLE
Input byte array has incorrect ending byte at 196

### ISSUE THREAD

**Issue by Ahmed-Elrayes** [laravel-idea/plugin#355]
Hello, laravel idea was going just fine but since yesterday i started to have error in php strom i will send the log in here so you might be able to identify it , thanks in advance
```
java.lang.IllegalArgumentException: Input byte array has incorrect ending byte at 196
	at java.base/java.util.Base64$Decoder.decode0(Base64.java:774)
	at java.base/java.util.Base64$Decoder.decode(Base64.java:538)
	at java.base/java.util.Base64$Decoder.decode(Base64.java:561)
	at com.laravel_idea.plugin.eloquent.c.d.complete(d.java:95)
	at com.jetbrains.php.PhpIndexImpl.lambda$doCompleteType$1(PhpIndexImpl.java:163)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:43)
	at com.jetbrains.php.PhpIndexImpl.doCompleteType(PhpIndexImpl.java:163)
	at com.jetbrains.php.PhpIndexImpl.completeType(PhpIndexImpl.java:120)
	at com.jetbrains.php.PhpIndexImpl.getClasses(PhpIndexImpl.java:240)
	at com.jetbrains.php.PhpIndexImpl.getBySignatureInternal(PhpIndexImpl.java:759)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:664)
	at com.jetbrains.php.PhpIndexImpl.getBySignature(PhpIndexImpl.java:651)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getBySignature(PhpExceptionAnalysisService.java:371)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.collectThrownExceptions(PhpExceptionAnalysisService.java:304)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.lambda$getPreventingRecursion$4(PhpExceptionAnalysisService.java:271)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:111)
	at com.intellij.openapi.util.RecursionGuard.doPreventingRecursion(RecursionGuard.java:43)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getPreventingRecursion(PhpExceptionAnalysisService.java:268)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getThrownExceptions(PhpExceptionAnalysisService.java:234)
	at com.jetbrains.php.lang.inspections.exception.PhpExceptionAnalysisService.getThrownExceptions(PhpExceptionAnalysisService.java:223)
	at com.jetbrains.php.lang.inspections.PhpThrownExceptionsAnalyzer.getExceptionClasses(PhpThrownExceptionsAnalyzer.java:76)
	at com.jetbrains.php.lang.inspections.probablyBug.PhpUnhandledExceptionInspection$Companion.getExceptionClasses(PhpUnhandledExceptionInspection.kt:142)
	at com.jetbrains.php.lang.inspections.probablyBug.PhpUnhandledExceptionInspection$buildVisitor$1.analyze(PhpUnhandledExceptionInspection.kt:65)
	at com.jetbrains.php.lang.inspections.probablyBug.PhpUnhandledExceptionInspection$buildVisitor$1.visitPhpMethodReference(PhpUnhandledExceptionInspection.kt:52)
	at com.jetbrains.php.lang.psi.elements.impl.MethodReferenceImpl.accept(MethodReferenceImpl.java:44)
	at com.jetbrains.php.lang.psi.elements.impl.PhpASTElementImpl.accept(PhpASTElementImpl.java:26)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:64)
	at com.intellij.codeInspection.InspectionEngine.createVisitorAndAcceptElements(InspectionEngine.java:55)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.runToolOnElements(LocalInspectionsPass.java:326)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$4(LocalInspectionsPass.java:285)
	at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:159)
	at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:151)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$5(LocalInspectionsPass.java:283)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:130)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:119)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$6(LocalInspectionsPass.java:283)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:136)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1078)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:92)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:104)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:83)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:183)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:83)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool.awaitJoin(ForkJoinPool.java:1708)
	at java.base/java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1039)
	at com.intellij.concurrency.JobLauncherImpl.invokeConcurrentlyUnderProgress(JobLauncherImpl.java:71)
	at com.intellij.concurrency.JobLauncher.invokeConcurrentlyUnderProgress(JobLauncher.java:49)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.visitPriorityElementsAndInit(LocalInspectionsPass.java:298)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.inspect(LocalInspectionsPass.java:210)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.collectInformationWithProgress(LocalInspectionsPass.java:121)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:56)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:399)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1078)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:392)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:705)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:647)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:63)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:391)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:367)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:183)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:365)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:188)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by adelf** [comment]
Thank you. I've fixed it. I'm going to release a new version next week.

**Comment by Ahmed-Elrayes** [comment]
Thanks a lot ♥

**Comment by adelf** [comment]
Fixed in the 4.4 version.
Full list of changes - https://laravel-idea.com/blog/2021/4-4-release

**Comment by Ahmed-Elrayes** [comment]
that is so great, ty
